lib/db.php

```diff
@@ -139,6 +139,7 @@ static public function connect($site_config) {
         try {
             self::$dbh[$key] = new PDO($dsn, self::$config['db_user'], self::$config['db_pass']);
             self::$dbh[$key]->setAttribute(PDO::ATTR_EMULATE_PREPARES, false);
+            self::$dbh[$key]->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
```

Comment:
I am afraid this will have more severe consequences and we will need to make sure to catch and handle all those possible exceptions from pdo queries.

Reply:
Yes, that is a temp change. And I would like to start a long term conversation about it
https://github.com/cypht-org/cypht/issues/977#issuecomment-2098904257

Reply:
I commented in #977